src/test/java/core/basesyntax/parsers/FruitStorageOperationsTest.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.parsers;
```

Comment:
this test class is placed in `parsers` folder
are you testing only classes from `parsers` folder here?